bin/si/README.md

```diff
@@ -42,34 +42,39 @@ key components:
 
 ```
 si
-├── ai-agent               - Manage the SI AI Agent (MCP server)
-│   ├── init               - Initialize AI agent configuration
-│   ├── start              - Launch Claude Code
-│   └── config             - Update configuration
-├── completion              - Generate shell completions
-├── component              - Component-related operations
-│   ├── get                - Get component data by name or ID
-│   ├── update             - Update component from JSON/YAML file
-│   ├── delete             - Delete a component
-│   └── search             - Search for components
-├── project
-│   └── init               - Initialize a new SI project
-├── remote
-│   └── schema
-│       ├── pull           - Pull schemas from remote workspace
-│       └── push           - Push schemas to remote workspace
-├── run                    - Run a SI template file
-├── schema
-│   ├── action generate    - Generate action functions
-│   ├── authentication generate - Generate authentication functions
-│   ├── codegen generate   - Generate code generator functions
-│   ├── management generate - Generate management functions
-│   ├── qualification generate - Generate qualification functions
-│   ├── scaffold generate  - Scaffold a complete schema
-│   └── overlay            - Generate overlay functions
-├── template
-│   └── run                - Run a SI template file (alias)
-└── whoami                 - Display authenticated user information
+├── ai-agent                          - Manage the SI AI Agent (MCP server)
+│   ├── init                          - Initialize AI agent configuration
+│   ├── start                         - Launch Claude Code
+│   ├── config                        - Update configuration
+│   └── stdio                         - Run MCP server in stdio mode (for external AI tools)
+├── completion                         - Generate shell completions
+├── component                         - Component-related operations
+│   ├── get                           - Get component data by name or ID
+│   ├── update                        - Update component from JSON/YAML file
+│   ├── delete                        - Delete a component
+│   └── search                        - Search for components
+├── schema                            - Manage schemas and project
+│   ├── init                          - Initialize a new SI project
+│   ├── generate                      - Generate schema functions
```

Comment:
This inverts the `si <noun> [<noun>...] <verb>` subcommand pattern, I would have expected `si schema action generate` (as was in the conduit codebase) as that allows us to add other interactions with action functions such as `si schema action edit`, `si schema action pull`, etc.